bin/fmt.sh

```diff
@@ -21,8 +35,11 @@ GOPATH=$(cd "$ROOTDIR/../../.."; pwd)
 export GOPATH
 export PATH=$GOPATH/bin:$PATH
 
-go get -u golang.org/x/tools/cmd/goimports
-goimports=${GOPATH}/bin/goimports
+if ! command -v goimports >/dev/null 2>&1; then
+  go get -u golang.org/x/tools/cmd/goimports
+fi
+
+goimports=$(command -v goimports)
```

Comment:
I'm a little concerned this could pick up the wrong goimports.  I am not a go toolchain wizard so not sure if ${GOPATH}/bin'/goimports is the conical location, but it seems the original author of this code thought that was the case.  If a different goimports were in the path, a potentially old version could be used.  This may be confusing.

Further I think the intent here was to always ensure goimports is up to date (via go get).  This is implied by the `-u` flag:

```
The -u flag instructs get to use the network to update the named packages
and their dependencies. By default, get uses the network to check out
missing packages but does not use it to look for updates to existing packages.
```

Reply:
Thanks for your review, this  make sense to me. I 
`rm -f /home/xsw/gowork/src/golang.org/x/tools` and rerun the script, the err has gone.

```
root@bz-ubuntu:/home/xsw/gowork/src/istio.io/istio# ./bin/fmt.sh
# cd /home/xsw/gowork/src/golang.org/x/tools; git pull --ff-only
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

package golang.org/x/tools/cmd/goimports: exit status 1
```
So We can close this.